eng/pipelines/templates/steps/update_snippet.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  ServiceDirectory: ''
+  ValidateFormatting: false
+  EnvVars: {}
+
+steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.7'
+    inputs:
+     versionSpec: '3.7'
+    condition: succeededOrFailed()
+
+  - task: PythonScript@0
+    displayName: 'Update Snippets'
+    inputs:
+      scriptPath: 'scripts/devops_tasks/python_snippet_updater.py'
```

Comment:
If you want to make this part of `azure-sdk-tools` under `ci_tools` as a tool, maybe you do that and add some unit tests so that you can be certain of this new code?

Reply:
Sorry, what's the difference between putting the script in this repo vs making it part of azure-sdk-tools?

Reply:
I'm really telling you

> I want tests for this feature

Reply:
Also created https://github.com/Azure/azure-sdk-tools/pull/4559 to add into sdk-tools repo.

Reply:
moved into azure-sdk-tools folder.